api/capi/src/nnstreamer-capi-util.c

```diff
@@ -960,6 +963,7 @@ ml_check_nnfw_availability (ml_nnfw_type_e nnfw, ml_nnfw_hw_e hw,
       }
       break;
     default:
+      goto done; /* Default = "Not available!" */
       break;
```

Comment:
This line will be never reached due to the goto statement.

Reply:
I'm not sure how SVACE/Coverity will react to this, but yes, let's remove break.